base/interactiveutil.jl

```diff
@@ -677,17 +677,20 @@ function whos(io::IO=STDOUT, m::Module=current_module(), pattern::Regex=r"")
             value = getfield(m, v)
             @printf head "%30s " s
             try
-                bytes = summarysize(value)
-                if bytes < 10_000
-                    @printf(head, "%6d bytes  ", bytes)
+                if s ∈ ("Base", "Main", "Core")
```

Comment:
I would do exactly `value ∈ (Base, Main, Core)`, so that it filters those exact items, and not something with the same name